### PR TITLE
Create 2022-05-30-openmic-workflow-announcement

### PR DESCRIPTION
**Pull request by ahmadjana**
post for next open mic

### CODE REVIEW

File: _posts/2022-05-30-openmic-workflow-announcement [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Upcoming Open Mic - workflow and tooling for front end applications"
+categories: [Open Mic Announcement, Open Mic]
+---
+
+The Open mic session starts on Friday 10-06-20202 at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart her presentation with the topic workflow and tooling for front end applications.
```
Comment (MichalMed) [inline _posts/2022-05-30-openmic-workflow-announcement:6]: this does not seem right 'Speaker Max Chopart her presentation???'

File: _posts/2022-05-30-openmic-workflow-announcement [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+###### Abstract:
+
+The presentation will be about Tooling and workflow for JavaScript libraries. The motivation is that when building a library (package) for another application also still under development, testing, integration and implementation can be rather tedious. Therefore, the aim is to provide some recommendations and guidelines to improve the CI and the workflow between development teams.
```
Comment (MichalMed) [inline _posts/2022-05-30-openmic-workflow-announcement:10]: some words are probably missing, does not make much sense. Suggestion:

The presentation is about Tooling and workflow for JavaScript libraries. Building libraries or packages used in another applications still under development may be tedious. Therefore the presentations aims form providing recommendations and guidelines  to improve CI and the workflow between development teams.

**Review by MichalMed — CHANGES_REQUESTED** [review]
Looks good, just improve language (see the comments).

File: _posts/2022-05-30-openmic-workflow-announcement (hunk 1 above)
Comment (MichalMed) [reply _posts/2022-05-30-openmic-workflow-announcement:6]: and the date does not seem right either. Try '10th June 2022'.

File: _posts/2022-05-30-openmic-workflow-announcement [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "Upcoming Open Mic - workflow and tooling for front end applications"
 categories: [Open Mic Announcement, Open Mic]
 ---
 
-The Open mic session starts on Friday 10-06-20202 at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart her presentation with the topic workflow and tooling for front end applications.
+The Open mic session starts on Friday '10th June 2022' at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart his presentation with the topic workflow and tooling for front end applications.
```
Comment (MichalMed) [inline _posts/2022-05-30-openmic-workflow-announcement:6]: ok, I appreciate change of his to her, but the sentence still does not make sense.

**Review by MichalMed — CHANGES_REQUESTED** [review]

File: _posts/2022-05-30-openmic-workflow-announcement (hunk 3 above)
Comment (ahmadjana) [reply _posts/2022-05-30-openmic-workflow-announcement]: hhh, yes. is it better now :)

**Review by MichalMed — APPROVED** [review]

**Pull request merged**